### DISCUSSION TITLE
Issue changing layoutConfig in Apollo v6

### DISCUSSION THREAD

**Discussion by ablyou** [primefaces/community#2966] [PrimeVue Templates]
Hello, 
I’m trying to update layoutConfig to customize the init theme settings in my app, like so : 

const layoutConfig = reactive({
    preset: 'Aura',
    primary: 'yellow',
    surface: 'ocean',
    darkTheme: true,
    menuMode: 'static',
    menuTheme: 'primaryColor'
});

However, the changes only seem to affect the menu, while the rest of the app remains unchanged :

![image](https://github.com/user-attachments/assets/6bc1309a-affe-4598-a2a3-bffe1dc43b12)
I know there has been some discussion about this issue, but I haven't found a working solution yet.

the flow : 
1) I downloaded the theme 
2) run 'npm install'
3) open the file and change the layoutConfig 

nothing else 

Thank you for your help.

**Comment by tugcekucukoglu** [comment]
Unfortunately, cannot replicate it.
